.nvmrc

```diff
@@ -0,0 +1 @@
+v18.15.0
```

Comment:
Do we need that, I feel like we're not gonna update this file, we're using engines so that people get an error or warning during installs

Reply:
I doesn't really affect me but we'll need to think about keeping that up to date